codechecker_common/output/codeclimate.py

```diff
@@ -22,18 +22,27 @@ def convert(reports: List[Report]) -> Dict:
     """
     codeclimate_reports = []
     for report in reports:
-        codeclimate_reports.append(__to_codeclimate(report))
+        codeclimate_reports.append(__to_codeclimate(report, severity_map))
     return codeclimate_reports
 
+__codeclimate_severity_map = {
+    'UNDEFINED': 'info',
+    'LOW': 'minor',
+    'STYLE': 'minor',
+    'MEDIUM': 'high',
+    'HIGH': 'critical',
```

Comment:
We have the following severity levels:
- Critical
- High
- Medium
- Low
- Style
- Unspecified

So I recommend to change this map to this one:
```suggestion
    'CRITICAL': 'critical',
    'HIGH': 'major',
    'MEDIUM': 'minor',
    'LOW': 'minor',
    'STYLE': 'info',
    'UNSPECIFIED': 'info'
```
What do you think?

Reply:
Sure, thanks. I have no idea where I got the `high` from.

Reply:
fixed